src/main/kotlin/com/mdrobnak/lalrpop/psi/LpResolveType.kt

```diff
@@ -16,7 +16,7 @@ import org.rust.lang.core.types.ty.TyUnit
 import org.rust.lang.core.types.type
 
 data class LpMacroArgument(val rustType: String, val name: String)
-data class LpMacroArguments(val arguments: List<LpMacroArgument> = listOf()) : List<LpMacroArgument> by arguments {
+data class LpMacroArguments(val rootArguments: List<LpMacroArgument>, val arguments: List<LpMacroArgument>) : List<LpMacroArgument> by arguments {
```

Comment:
Can you write a comment explaining the difference between `rootArguments` and `arguments`?